pkgs/default.nix

```diff
@@ -116,4 +116,63 @@ rec {
 
   deploymentModules = self.deploymentModules;
 
-} // modules
+  allModules = [
+    (import ./moduleit/module-definition.nix)
+    # (import ./modules/bundles/go)
+    # (import ./modules/compilers/go)
+    # (import ./modules/languageServers/gopls)
+    # (import ./modules/formatters/gofmt)
+    # (import ./modules/bundles/ruby)
+    # (import ./modules/interpreters/ruby)
+    # (import ./modules/languageServers/solargraph)
+    # (import ./modules/packagers/rubygems)
+    # (import ./modules/bundles/nodejs)
+    # (import ./modules/interpreters/nodejs)
+    # (import ./modules/formatters/prettier)
+    # (import ./modules/languageServers/typescript-language-server)
+    # (import ./modules/packagers/nodejs-packager)
+    # (import ./modules/debuggers/js-debug)
+    # (import ./modules/bundles/bun)
+    # (import ./modules/interpreters/bun)
+    # (import ./modules/packagers/bun)
+    # (import ./modules/bundles/web)
+    # (import ./modules/languageServers/css-language-server)
+    # (import ./modules/languageServers/html-language-server)
+  ];
+
+  myEvalModule = path:
+    (pkgs.lib.evalModules {
+      modules = [
+        (import path)
+      ] ++ allModules;
+      specialArgs = {
+        inherit pkgs pkgs-23_05;
+        pkgs-unstable = pkgs;
+        modulesPath = builtins.toString ./.;
```

Comment:
does this need to be a string? can't we pass the path directly? i know there's no *real* difference between the 2 but it feels better to be passing a path than a string

Reply:
Okay. I also changed it to the modules directory. We are not using it anywhere though.